// Exercice 2: GÃ©rer les erreurs avec try...catch
